recipes/hidefix/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "hidefix" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hidefix-{{ version }}.tar.gz
+  sha256: 9ac6c81783e1d8c401c4c5bd2d2df1f6f85bc91de32889d54acf62506bbd6910
+
+build:
+  number: 0
```

Will need to run `cargo-bundle-licenses` also. Do you prefer to add it in `recipe/meta.yaml` or in the `build.sh`/`bld.bat` script?

Option 1 is to do this, similar to https://github.com/conda-forge/lazrs-python-feedstock/blob/9954d7461d4a054855bd3f351d7d3d83cf4b2eb4/recipe/meta.yaml#L12-L15, but that means you'll lose the extra configuration in bld.bat/build.sh:

```suggestion
build:
  number: 0
  script:
    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
    - {{ PYTHON }} -m pip install . -vv
```

Option 2 is to add it to `build.sh` and `bld.bat`, similar to https://github.com/conda-forge/polars-feedstock/blob/287e41db502984c1fe6528dded5aa1f57bea20d4/recipe/build.sh#L14.

---

If the build/bld files are not necessary that would be the best! I don't think I managed to get it to work without them, but maybe that has changed now?

---

You could try to delete the build.sh/bld.bat files and commit the above change as a test? Can always revert back if needed.

---

> You could try to delete the build.sh/bld.bat files and commit the above change as a test? Can always revert back if needed.

I just tried compiling locally on Linux without the `build.sh` file, just the script commands above in `recipe/meta.yaml`, and it seemed to work. Not sure about macOS or Windows though.

---

Trying it now. The feature configuration stuff has been moved into pyproject.toml.